cmd/kubectl-gadget/audit/seccomp.go

```diff
@@ -81,11 +81,17 @@ func auditSeccompTransformLine(line string) string {
 	}
 
 	switch params.OutputMode {
+	case commonutils.OutputModeJSON:
+		b, err := json.Marshal(e)
+		if err != nil {
+			fmt.Fprint(os.Stderr, fmt.Sprint(commonutils.WrapInErrMarshalOutput(err)))
+			return ""
+		}
+		sb.WriteString(string(b))
```

Comment:
I suppose this change is needed because we still don't have a parser implemented for this gadget, right?

Reply:
Exactly. I implemented the parser for that gadget in https://github.com/kinvolk/inspektor-gadget/pull/770.